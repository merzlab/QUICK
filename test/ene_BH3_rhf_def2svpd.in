HF  BASIS=def2-svpd cutoff=1.0e-9 denserms=1.0e-6  zmake ENERGY DIPOLE CHARGE=0

B         -1.80396        1.16364       -0.15206
H         -0.70396        1.16364       -0.15206
H         -2.35396        0.28979       -0.53138
H         -2.35396        2.03749        0.22726
