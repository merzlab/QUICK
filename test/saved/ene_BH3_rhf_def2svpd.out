  *********************************************************************************************************
  **                                                                                                     **
  **           888888888888                                                                              **
  **         8888888888888888                                                                            **
  **      888888888888888888                                                                             **
  **     888888888888888888                                                                              **
  **    888888888888888                                                                                  **
  **   88888888888888888888                               88888                       8888:              **
  **   8888888888888888888888Z                            88888                       8888:              **
  **   888888888888888888888888?                          88888                       8888:              **
  **   8888888888888      8888888                                        888888       8888:              **
  **    88888888888         888888     8888:     88888    88888        888888888I     8888:    888888    **
  **    8888888888           88888:    8888:     88888    88888      $888888888888    8888:   888888     **
  **    I8Z 88888             88888    8888:     88888    88888    .888888     8888   8888: 888888       **
  **    .8Z 88888             88888    8888:     88888    88888    $88888             8888:88888         **
  **     8I 88888      .=88. .88888    8888:     88888    88888    88888              8888888888         **
  **    :8  88888      888888$8888$    8888:     88888    88888    8888O              88888888888        **
  **   ,7   +88888.     8888888888.    8888:     88888    88888    888888             88888O888888       **
  **         $888888:.   .8888888      88888....888888    88888     888888     8888   8888:  888888      **
  **          I8888888888888888888     888888888888888    88888     O888888888888O    8888:   888888     **
  **            O888888888888888888     88888888888888    88888       88888888888$    8888:    888888    **
  **               8888888Z     888      .8888I  88888    88888         8888888       8888:     888888   **
  **                                                                                                     **
  **                                                                                                     **
  **                                         Copyright (c) 2021                                          **
  **                          Regents of the University of California San Diego                          **
  **                                    & Michigan State University                                      **
  **                                        All Rights Reserved.                                         **
  **                                                                                                     **
  **                   This software provided pursuant to a license agreement containing                 **
  **                   restrictions on its disclosure, duplication, and use. This software               **
  **                   contains confidential and proprietary information, and may not be                 **
  **                   extracted or distributed, in whole or in part, for any purpose                    **
  **                   whatsoever, without the express written permission of the authors.                **
  **                   This notice, and the associated author list, must be attached to                  **
  **                   all copies, or extracts, of this software. Any additional                         **
  **                   restrictions set forth in the license agreement also apply to this                **
  **                   software.                                                                         **
  *********************************************************************************************************
                                                                                                           
  Cite this work as:                                                                                       
  Manathunga, M.; Chi, J.; Cruzeiro, V.W.D.; Keipert, K.; Pekurovsky, D.; Mu, D.; Miao, Y.;He, X.; Ayers,K;
  Brothers, E.; GÃ¶tz, A.; Merz,K. M. QUICK-21.03 University of California San Diego, CA and Michigan State
  University, East Lansing, MI, 2021                                                                       
                                                                                                           
  If you have any comments or queries, please send us an email for technical support:                      
  quick.merzlab@gmail.com                                                                                  
                                                                                                           
| TASK STARTS ON: Thu Mar  4 18:16:37 2021
| INPUT FILE :    ene_BH3_rhf_def2svpd.in
| OUTPUT FILE:    ene_BH3_rhf_def2svpd.out
| DATE FILE  :    ene_BH3_rhf_def2svpd.dat
| BASIS SET PATH: /home/mmanathunga/fix109/basis

 @ Read Job And Atom

  KEYWORD=HF  BASIS=DEF2-SVPD CUTOFF=1.0E-9 DENSERMS=1.0E-6  ZMAKE ENERGY DIPOLE CHARGE=0                                                                                                                         
 ============== JOB CARD =============
 METHOD = HATREE FOCK
 DIRECT SCF 
 SAD INITAL GUESS 
 USE DIIS SCF
 PRINT ENERGY EVERY CYCLE
 Z-MATRIX CONSTRUCTION
 DIPOLE
 MAX SCF CYCLES =    200
 MAX DIIS CYCLES =   10
 DELTA DENSITY START CYCLE = 1000
 COMPUTATIONAL CUTOFF: 
      TWO-e INTEGRAL   =  0.100E-08
      BASIS SET PRIME  =  0.100E-19
      MATRIX ELEMENTS  =  0.100E-09
      BASIS FUNCTION   =  0.100E-09
 DENSITY MATRIX MAXIMUM RMS FOR CONVERGENCE  =  0.100E-05
 BASIS SET = DEF2-SVPD,  TYPE = CARTESIAN
| BASIS FILE = /home/mmanathunga/fix109/basis/DEF2-SVPD.BAS

 @ Finish reading job


 @ Begin SAD initial guess

 For Atom Kind =    1
 ELEMENT = B 
 BASIS FUNCTIONS =   22
 For Atom Kind =    2
 ELEMENT = H 
 BASIS FUNCTIONS =    8

 @ Finish SAD initial guess


 @ Begin Reading Molecular Information


 =========== Molecule Input ==========
 TOTAL MOLECULAR CHARGE  =    0    MULTIPLICITY                =    1
 TOTAL ATOM NUMBER       =    4    NUMBER OF ATOM TYPES        =    2
 NUMBER OF HYDROGEN ATOM =    3    NUMBER OF NON-HYDROGEN ATOM =    1
 NUMBER OF ELECTRONS     =    8

 -- INPUT GEOMETRY -- :
    B          -1.8040       1.1636      -0.1521
    H          -0.7040       1.1636      -0.1521
    H          -2.3540       0.2898      -0.5314
    H          -2.3540       2.0375       0.2273

 -- DISTANCE MATRIX -- :
              1           2           3           4
      1     0.00000
      2     2.07870     0.00000
      3     2.07870     3.60041     0.00000
      4     2.07870     3.60041     3.60041     0.00000

============== BASIS INFOS ==============
 BASIS FUNCTIONS =   46
 NSHELL =   20 NPRIM  =   62
 JSHELL =   20 JBASIS =   32


 @ End Reading Molecular Information 


 @ Begin Energy Calculation

                                         SCF ENERGY
------------------------------------------------------------------------------------------------------------------------
NCYC       ENERGY         DELTA_E      SCF_TIME  DII_CYC   DII_TIME   O_TIME  DIAG_TIME    MAX_ERR    RMS_CHG    MAX_CHG
------------------------------------------------------------------------------------------------------------------------
|  1    -26.486705640      ------         0.087     1        0.00      0.09      0.00    0.1413E+01  0.3284E-01  0.3116E+00
|  2    -26.340342132  -.146364E+00       0.089     2        0.00      0.09      0.00    0.5501E-01  0.6987E-02  0.1082E+00
|  3    -26.355406091  0.150640E-01       0.085     3        0.00      0.08      0.00    0.1103E-01  0.3364E-02  0.5405E-01
|  4    -26.356238273  0.832181E-03       0.085     4        0.00      0.08      0.00    0.2537E-02  0.6462E-03  0.9606E-02
|  5    -26.356272877  0.346041E-04       0.085     5        0.00      0.08      0.00    0.9302E-03  0.1335E-03  0.1398E-02
|  6    -26.356276961  0.408395E-05       0.085     6        0.00      0.08      0.00    0.3430E-03  0.4177E-04  0.4887E-03
|  7    -26.356277458  0.497026E-06       0.085     7        0.00      0.08      0.00    0.7470E-04  0.9446E-05  0.9119E-04
|  8    -26.356277484  0.260243E-07       0.085     8        0.00      0.08      0.00    0.1058E-04  0.3413E-05  0.4762E-04
|  9    -26.356277484  0.630946E-09       0.085     9        0.00      0.08      0.00    0.8722E-06  0.3280E-06  0.3599E-05
------------------------------------------------------------------------------------------------------------------------
| REACH CONVERGENCE AFTER   9 CYLCES
| MAX ERROR = 0.872202E-06   RMS CHANGE = 0.328045E-06   MAX CHANGE = 0.359873E-05
 -----------------------------------------------
 ELECTRONIC ENERGY    =   -34.405573916
 CORE_CORE REPULSION  =     8.049296432
 TOTAL ENERGY         =   -26.356277484

 @ End Energy calculation


 @ Begin Zmake conversion

  
 Z-MATRIX:

   ATOM  ELEMENTAL  BOND LENGTH    BOND ANGLE    DIHEDRAL ANGLE
  NUMBER  SYMBOL    (ANGSTROMS)    (DEGREES)       (DEGREES)
     I                 I-NA         I-NA-NB        I-NA-NB-NC      NA   NB   NC

     1      B        0.00000  0     0.00000  0      0.00000  0      0    0    0
     2      H        1.10000  1     0.00000  0      0.00000  0      1    0    0
     3      H        1.10000  1   120.00003  1      0.00000  0      1    2    0
     4      H        1.10000  1   120.00003  1   -180.00000  1      1    2    3

 @ End Zmake conversion


 @ Begin Charge and Dipole Calculation

    ATOMIC CHARGES
   ATOM          MULLIKEN            LOWDIN
   B             -0.3840             0.1014
   H              0.1280            -0.0440
   H              0.1280            -0.0287
   H              0.1280            -0.0287
   TOTAL         0.0000             0.0000

    DIPOLE (DEBYE)
      X         Y         Z        TOTAL
   -0.0000   -0.0000   -0.0000    0.0000

 @ End Charge and Dipole Calculation


 @ Output Timing Information

------------- TIMING ---------------
| INITIAL GUESS TIME  =     4.062720000( 82.31%)
| TOTAL SCF TIME      =     0.776839000( 15.74%)
|       TOTAL OP TIME      =     0.768148000( 15.56%)
|             TOTAL 1e TIME      =     0.006333000(  0.13%)
|             TOTAL 2e TIME      =     0.755270000( 15.30%)
|       TOTAL DII TIME      =     0.008547000(  0.17%)
|             TOTAL DIAG TIME    =     0.003087000(  0.06%)
| TOTAL TIME          =     4.935625000
------------------------------------
| Job cpu time:  0 days  0 hours  0 minutes  4.9 seconds.

 @ Finish Output Timing Information

| Normal Termination. Task Finished on: Thu Mar  4 18:16:42 2021
